src/components/ShowMetadata.jsx

```diff
@@ -4,7 +4,7 @@ import Divider from '../images/img_divider.png';
 
 const ShowMetadata = ({ metadata }) => {
   return (
-    <div className="show-metadata">
+    <div className="handle show-metadata">
```

Comment:
The `.handle` class was added to several components in order to avoid the "drag while typing" bug that appeared with the selected annotation input box. In this case, all children of a `Dialog` component are given the `.handle` class if they need to drag, where the `CribEdit` component toggles this on and off depending on if the input box is selected. 

Without the introduction of this `dragHandlerClassName` (see `Dialog` component), any clickable space in a `Dialog` component (including children) would initiate a drag.